app/assets/stylesheets/chat.scss

```diff
@@ -165,7 +165,8 @@
   right: 0;
   bottom: 0;
   transform: translateX(0);
-  transition: transform 0.35s linear;
+  transition: transform 0.22s linear;
+  z-index: var(--z-elevate);
```

Comment:
This is a z-index bug fix that is unrelated to the animation, but still a bug fix.